libraries/src/Changelog/Changelog.php

```diff
@@ -26,8 +27,10 @@
  *
  * @since  4.0.0
  */
-class Changelog extends CMSObject
+class Changelog
 {
+    use LegacyErrorHandlingTrait;
+    use LegacyPropertyManagementTrait;
```

Comment:
```suggestion
    use LegacyPropertyManagementTrait;

```
Fix PHPCS failing in Drone.